src/ILCompiler/reproNative/reproNative.vcxproj

```diff
@@ -20,7 +20,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <!-- v140 is friendly to windbg, at the moment -->
+    <PlatformToolset>v140</PlatformToolset>
```

Comment:
Changes like this should ideally be done in a separate pull request instead of sneaking them in a big change.